.github/workflows/backport.yml

```diff
@@ -5,7 +5,7 @@ on:
 jobs:
   backport:
     name: Backport Pull Request
-    if: github.repository_owner == 'NixOS' && github.event.pull_request.merged == true
+    if: github.repository_owner == 'NixOS' && github.event.pull_request.merged == true && (github.event_name != 'labeled' || startsWith('backport', github.event.label.name))
```

Comment:
Shouldn't it also be triggered by a merge event?

Reply:
`github.event_name != 'labeled'` will be `true` then

Reply:
Oops, I misread it.